lte/gateway/c/core/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -3336,6 +3334,13 @@ void mme_app_handle_handover_required(
         ue_context_p->emm_context._security.vector_index,
         ue_context_p->mme_ue_s1ap_id);
   }
+
+  /* TODO: investigate
+     1- Should we bump next_hop and use it in this message (as is)
+     2- or, use the previously calculated next_hop and bump up for next
+        iteration. In this case, we are missing a bump to next_hop someewhere
+        else, per testing
+  */
```

Comment:
TL;DR We should bump up next_hop and ncc, store these new values and send them to target eNB in S1 Handover Request.

https://www.etsi.org/deliver/etsi_ts/133400_133499/133401/15.07.00_60/ts_133401v150700p.pdf section 7.2.8.4.3 indicates that (source and target MMEs in this case are the same):
- "Upon reception of the HANDOVER REQUIRED message the source MME shall increase its locally kept NCC value
by one and compute a fresh NH from its stored data" 
- "The source MME shall store that fresh pair and send it to the target MME in the S10 FORWARD RELOCATION REQUEST message." (here S10 can be ignored as source and target MMEs are the same.
- "The target MME shall store locally the {NH, NCC} pair received from the source MME.The target MME shall then send the received {NH, NCC} pair to the target eNB within the S1 HANDOVER REQUEST."